### ISSUE TITLE
React js version 18 typescripts issue

### ISSUE THREAD

**Issue by kkrajpurohit007** [maxmantz/redux-oidc#202]
### Issue
TS2769: No overload matches this call
```
    35 |   const root = ReactDOM.createRoot(document.getElementById("root")!);
    36 |   root.render(
  > 37 |     <OidcProvider store={reduxStore} userManager={usrMgr as any}>
       |      ^^^^^^^^^^^^
    38 |       <Provider store={reduxStore}>
    39 |         <GlobalStyle />
    40 |         <App />
```
### Solution
export interface OidcProviderProps<TSTate> {
    readonly userManager: UserManager;
    readonly store: Store<TSTate>;
    **children:React.ReactNode;**
  }
  
  
  We need change urgently. Can team implement these?

**Comment by drod21** [comment]
Hey @kkrajpurohit007 I recently took over as maintainer, working on updating this project and figuring out how to publish/deploy, but in the meantime, you should be able to use redux-oidc-hooks from npm which is a fork with this update published

**Comment by ryanzhangau** [comment]
Hi @drod21 ,

I think we have merged the fix for this issue. Just need to provide a new tag version for installation.
Can you please kindly help us with the version.

Many thanks

Ryan

**Comment by drod21** [comment]
@ryanzhangau the tag is https://github.com/maxmantz/redux-oidc/releases/tag/4.0.0-beta2

but i'm also not able to release this to npm myself, i just published a package called redux-oidc-js version 0.0.1 that has this fix in while i wait to see if i can take over the npm package as well.

redux-oidc-js is published off of this repo, i can update the npm install instructions in the meantime.

**Comment by mk0sojo** [comment]
I think something has gone wrong with the redux-oidc-js package. Installing it and looking at package.json it has "react": "^16.8.4" (and few more packages) in dependencies. . This creates warnings when installing with npm 8.15 and React 18.
